EIPS/eip-hard-gas-target.md

```diff
@@ -0,0 +1,60 @@
+---
+eip: xxxx
+title: Hardcoded Block Gas Target
+author: Philippe Castonguay (@PhABC)
+discussions-to: xxxx
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-03-13
+
+---
+
+## Simple Summary
+
+Hardcode the block gas target (previously known as *block gas limit* before [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md)) to `12,500,000` gas per block.
+
+## Abstract
+
+Ethereum's block gas target is currently dictated by block producers and is not enforced when validating blocks in clients. This EIP proposes to hardcode the block gas target to `12,500,000`.
+
+## Motivation
+
+Both Ethereum's Proof of Work and Proof of Stake designs assume that block producers are financially rational, but does not assume block producers to be benevolent. There is one exception however, and it is when block producers choose the gas targe of a block (previously known as block gas limit), where it is assumed that block producers care about the long term health and decentralisation of the chain. Indeed, the block gas target is one of the only parameters in Ethereum that is not dictated by node consensus, but instead is chosen by block producers. This decision was initially made to allow urgent changes in the block gas target if necessary. Both drastically increasing or decreasing this parameter could have serious consequences that may not be desired. It is therefore a critical parameter that should require node consensus to avoid any sudden harmful change imposed by a small number of actors on the rest of the network.
```

Comment:
I'll need to check later if the standard easy of referring to POW and POS is hyphenated or not.

Reply:
Searched and found a few EIPs that use these terms the same way, e.g. https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1011.md and https://eips.ethereum.org/EIPS/eip-1057#motivation while others do used an hyphenated version like https://eips.ethereum.org/EIPS/eip-3368 and  https://eips.ethereum.org/EIPS/eip-1057's title.

Reply:
Doesn't appear to be a strong consensus on one version or the other. Please use your preferred. If certain style is decided on the future, all EIPs will be updated accordingly.

```console
$ grep -ril "proof of work" EIPS | wc -l
9
$ grep -ril "proof-of-work" EIPS | wc -l
13
$ grep -ril "proof of stake" EIPS | wc -l
5
$ grep -ril "proof-of-stake" EIPS | wc -l
8
```

Reply:
Thanks for doing the search, looks pretty evenly distributed!